operator_ui/src/pages/Jobs/New.tsx

```diff
@@ -156,10 +156,8 @@ export const New = ({
             setServerErrorMsg(error.toString())
           }
 
-          setValid(false)
-        })
-        .finally(() => {
           setLoading(false)
+          setValid(false)
```

Comment:
Why this change? It seems like only doing `setLoading(false)` on error would not disable loading state on success?

Reply:
This was part of my debugging of tests that I was meant to come back to. Thanks for picking that up